website/pages/docs/plugins/destinations/neo4j/types.md

```diff
@@ -0,0 +1,48 @@
+# Neo4j Types
+
+The Neo4j destination (`v3.0.0` and later) supports most [Apache Arrow](https://arrow.apache.org/docs/index.html) types. The following table shows the supported types and how they are mapped to [Neo4j data types](https://neo4j.com/docs/graphql-manual/current/type-definitions/types/).
+
+| Arrow Column Type      | Supported?  | Neo4j Type |
+|------------------------|-------------|------------|
+| Binary                 | ✅ Yes      | `Bytes`    |
+| Boolean                | ✅ Yes      | `Boolean`  |
+| Date32                 | ✅ Yes      | `String`   |
+| Date64                 | ✅ Yes      | `String`   |
+| Decimal                | ✅ Yes      | `String`   |
+| Dense Union            | ✅ Yes      | `String`   |
+| Dictionary             | ✅ Yes      | `String`   |
+| Duration[ms]           | ✅ Yes      | `String`   |
+| Duration[ns]           | ✅ Yes      | `String`   |
+| Duration[s]            | ✅ Yes      | `String`   |
+| Duration[us]           | ✅ Yes      | `String`   |
+| Fixed Size List        | ✅ Yes      | `String`   |
+| Float16                | ✅ Yes      | `String`   |
+| Float32                | ✅ Yes      | `Float`    |
+| Float64                | ✅ Yes      | `Float`    |
+| Inet                   | ✅ Yes      | `String`   |
+| Int8                   | ✅ Yes      | `BigInt`   |
+| Int16                  | ✅ Yes      | `BigInt`   |
+| Int32                  | ✅ Yes      | `BigInt`   |
+| Int64                  | ✅ Yes      | `BigInt`   |
+| Interval[DayTime]      | ✅ Yes      | `String`   |
+| Interval[MonthDayNano] | ✅ Yes      | `String`   |
+| Interval[Month]        | ✅ Yes      | `String`   |
+| JSON                   | ✅ Yes      | `String`   |
+| Large Binary           | ✅ Yes      | `Bytes`    |
+| Large List             | ✅ Yes      | `List`     |
+| Large String           | ✅ Yes      | `String`   |
+| List                   | ✅ Yes      | `List`     |
+| MAC                    | ✅ Yes      | `String`   |
+| Map                    | ✅ Yes      | `String`   |
+| String                 | ✅ Yes      | `String`   |
+| Struct                 | ✅ Yes      | `String`   |
```

Comment:
Should we document how the struct is actually encoded into string?

Reply:
Yes maybe https://github.com/cloudquery/cloudquery/pull/10955